tests/tools/test_line_cache.py

```
@@ -0,0 +1,181 @@
+            line_number=offset + i + 1,
+            hash=content_hash(c),
+            content=c,
+        )
+        for i, c in enumerate(contents)
+    ]
+
+
+class TestStoreAndGet:
+    def test_store_and_retrieve(self) -> None:
+        lines = _make_lines(["a", "b", "c"])
+        line_cache.store("/tmp/test.py", lines)
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/test.py"

(S108)